roles/hypervisor/files/prepare.sh

```diff
@@ -15,21 +15,28 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-
+set -eux
 openstack_nodes="openstack1 openstack2 openstack3"
-image_path="/var/lib/libvirt/images/"
+image_path="/var/lib/libvirt/images"
+edeploy_path="/srv/edeploy"
+role_version="D7-I.1.3.0"
```

Comment:
ansible-spinalstack is not edeploy related. Some user are using it with qcow2 generated via disk-image-builder, so `role_version` and `edeploy_path` shouldn't hang around